TombEngine/Scripting/Internal/TEN/Strings/DisplayString/DisplayString.cpp

```diff
@@ -132,7 +132,12 @@ void DisplayString::Register(sol::table & parent)
 		// @function DisplayString:GetPosition
 		// @treturn int x x-coordinate of the string
 		// @treturn int y y-coordinate of the string
-		ScriptReserved_GetPosition, &DisplayString::GetPos
+		ScriptReserved_GetPosition, &DisplayString::GetPos,
+
+		/// Set the display string's flags 
+		// @function DisplayString:SetFlags
+		// @tparam Table table the new table with display flags options
+		ScriptReserved_SetFlags, & DisplayString::SetFlags
```

Comment:
Please add an example just to make it clear that the `Strings.DisplayStringOption` flag should be used (assuming they are)